app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    name_list = command.split(" ")
+    if name_list[1] != name_list[2]:
+        with open(name_list[1], "r") as file_get:
+            info = file_get.read()
+        with open(name_list[2], "w") as file_write:
+            file_write.write(info)
```

Comment:
use the hint 


![image](https://user-images.githubusercontent.com/104218550/211187698-d8343607-ba33-4190-af53-5b4f722ed3b6.png)